### ISSUE TITLE
allow empty functions option does not work

### ISSUE THREAD

**Issue by yokots** [palantir/tslint#4405]
### Bug Report

- __TSLint version__: 5.12.0
- __TypeScript version__: 3.2.2
- __Running TSLint via__: CLI / VSCode TSLint Plugin

#### TypeScript code being linted

```ts
// code snippet
export function noop() {}
```

with `tslint.json` configuration:

```json
{"no-empty": [true, "allow-empty-catch", "allow-empty-functions"]}
```

#### Actual behavior
allow empty functions option does not work

**Comment by johnwiseheart** [comment]
I can't reproduce this on my computer. Just to confirm, your full tslint configuration probably looks something closer to 
```json
{
  "defaultSeverity": "error",
  "extends": ["tslint:recommended"],
  "jsRules": {},
  "rules": {
    "no-empty": [true, "allow-empty-catch", "allow-empty-functions"]
  },
  "rulesDirectory": []
}
```

right?

**Comment by yokots** [comment]
Sorry, i found that vrsrouce-rules rewrite my rules.
